data_models/README.md

```diff
@@ -136,8 +136,48 @@ The fields will be used for normalizing the county case and death tabulations, a
         }
     }
 ```
-5. __Hospitalization Data__  
-We plan to retrieve hospitalization data by county from a common source, the [California Health and Human Services Open Data Portal (CHHS)](https://data.chhs.ca.gov/dataset/california-covid-19-hospital-data-and-case-statistics/resource/6cd8d424-dfaa-4bdd-9410-a3d656e1176e?inner_span=True.). These datapoints are not currently reflected in the data model.
+5. __Hospitalization Data__
+California COVID-19 hospitalization data is retrieved separately from the the [California Health and Human Services Open Data Portal (CHHS)](https://data.ca.gov/dataset/covid-19-hospital-data#). The keys for the top-level data points `name`, `update_time`, `source_url`, `meta_from_source`, and `meta_from_baypd` are the same as the model described above. However, the structure of the data for `meta_from_source` and `series` differs.
+
+`meta_from_source` holds a list of dicts describing each field in the data, provided directly from CHHS. Each dict, except for the first describing the `_id` field, has the keys `info`, `type`, and `id`. Here is an example:
+
+```
+    "meta_from_source": [
+        {
+            "type": "int",
+            "id": "_id"
+        },
+        {
+            "info": {
+                "notes": "The County where the hospital is located. None of the consolidated reporters had hospitals in different counties.",
+                "type_override": "",
+                "label": "County"
+            },
+            "type": "text",
+            "id": "county"
+        }
+
+        -- snip --
+```
+
+Each entry in `series` is a flat record with each of the fields described in `meta_from_source`, one for each day on which an observation was made:
+
+```
+        {
+            "icu_covid_confirmed_patients": 0.0,
+            "icu_suspected_covid_patients": 0.0,
+            "hospitalized_covid_patients": null,
```

Comment:
Please replace `null` values with `-1`, per [the data README](https://github.com/sfbrigade/data-covid19-sfbayarea/blob/master/data_models/README.md#non-number-values)
(By the way! We have _another_ README in `./data_models` just for data sleuths like yourself!)

I am also tempted to ask that you cast all integer values to `int`s so they appear as whole numbers and throw errors otherwise, since that is what we are doing with other integer data. At the very least, it will help the front end be consistent if they use tooltips to read off values. Let me know if you see a reason to keep the precision to a 1/10th.

Reply:
Got it. All of the data in the fields I've seen so far (except for `rank`) are either whole numbers or null, so casting as `int` makes sense. I'll follow up if I find it's throwing lots of errors.

Reply:
What is `rank`? I just noticed that. You don't mention it in the README additions nor in `hospitals_data_model.json`. Based on a quick visual inspection of Marin's timeseries, it looks like it's the same value for every day.

Do you know what the `rank` column is - if so, can you add descriptions to the README and hospitals data model? If not, I'm inclined to skip the column for now. You can flag it in the README for the hospitalization section, and also link to a new issue to resolve it when we know more.

Reply:
That field appears to be undocumented, and so I can only guess as to what it is. I'll remove it from the data, and as you suggest, open a new issue and make a note in the README.